[#61332] 15.3 Recurring meetings and meetings changes  https://commun…

MayaBerd opened the pull request:
…ity.openproject.org/work_packages/61332

[#61332] 15.3 Recurring meetings and meetings changes

https://community.openproject.org/work_packages/61332

# Ticket
<!-- Provide the link to respective work package -->

<!-- Contributors: Please check our PR guide: https://www.openproject.org/docs/development/code-review-guidelines/#preparing-your-pull-request before opening a PR. -->

<!-- Reviewers: Please check our Review guide: https://www.openproject.org/docs/development/code-review-guidelines/#reviewing -->

# What are you trying to accomplish?
<!-- Provide a description of the changes. -->

## Screenshots
<!-- Provide before/after screenshots, videos, or graphs for any visual changes; otherwise, remove this section -->

# What approach did you choose and why?
<!-- This section is a place for you to describe your thought process in making these changes.
     List any tradeoffs you made to take on or pay down tech debt.
     Describe any alternative approaches you considered and why you discarded them. -->

# Merge checklist

- [ ] Added/updated tests
- [ ] Added/updated documentation in Lookbook (patterns, previews, etc)
- [ ] Tested major browsers (Chrome, Firefox, Edge, ...)

MayaBerd committed:
[#61332] 15.3 Recurring meetings and meetings changes  https://community.openproject.org/work_packages/61332

[#61332] 15.3 Recurring meetings and meetings changes

https://community.openproject.org/work_packages/61332

MayaBerd committed:
add docs-on one-time meetings

MayaBerd committed:
[#61332] 15.3 Recurring meetings and meetings changes  https://community.openproject.org/work_packages/61332

[#61332] 15.3 Recurring meetings and meetings changes

https://community.openproject.org/work_packages/61332

MayaBerd committed:
Update README.md

MayaBerd commented:
@as-op I've destroyed quite a few links with the meetings docs change, could you please take a look them? We would need a redirect from the removed Dynamic Meetings (https://www.openproject.org/docs/user-guide/meetings/dynamic-meetings/) to overall Meetings docs page (https://www.openproject.org/docs/user-guide/meetings/). Thank you!

as-op committed:
adjust links

as-op reviewed (approved):
Links are adjusted. 
I removed two future warnings about the dynamic meetings to be removed, as they are now removed.

psatyal committed:
Update README.md

Minor changes to phrasing

psatyal committed:
Minor

psatyal committed:
Update README.md

Changed "the user" or "a user" to the direct second person "you" to make it consistent with the language preceding the list.

psatyal committed:
Update README.md

Minor update to specify that dynamic meetings aren't being replaced by one-time or recurring meetings, but that those are different types of dynamic meetings.

psatyal committed:
Changes for accuracy and more details

Also updated to take into account the changes to the phrasing of meeting series frequency that will be introduced by https://community.openproject.org/wp/61476

psatyal reviewed (approved):
Hi @MayaBerd. I've reviewed all edited docs pages and made a few changes where necessary.

psatyal committed:
minor

MayaBerd commented on docs/user-guide/meetings/recurring-meetings/README.md:
```diff
@@ -0,0 +1,115 @@
+
+This will open the specific meeting page. You can then edit the meeting by using same functions as for [editing one-time meetings](../one-time-meetings), including adding sections and agenda points, inviting participants, and adding attachments. 
+
+Additionally you also copy a specific meeting series occurrence as a one-time meeting. To do that click the **More ** (three dots) icon and select **Copy as one-time meeting**.
```
```suggestion
Additionally you can copy a specific meeting series occurrence as a one-time meeting. To do that click the **More ** (three dots) icon and select **Copy as one-time meeting**.
```

MayaBerd committed:
Update docs/user-guide/meetings/recurring-meetings/README.md

MayaBerd committed:
Merge branch 'release/15.3' into documentation/61332-153-recurring-meetings-and-meetings-changes

MayaBerd committed:
fix a heading name
